tutorials/social-recovery/blockchain_app/test/simulate_recovery.js

```diff
@@ -0,0 +1,189 @@
+const { createTransferTrx } = require('./utils/transactions/balance_transfer');
+const { createRecoveryConfigTrs } = require('./utils/transactions/create_recovery_trx');
+const { createInitiateRecoveryTrx } = require('./utils/transactions/initiate_trx');
+const { createVouchRecoveryTrx } = require('./utils/transactions/vouch_trx');
+const { getIPCClient } = require('./utils/api_client');
+const { default: axios } = require('axios');
+
+const getURL = (url, port = 4000) => `http://localhost:${port}${url}`;
+
+const wait = async (timeInMS) => new Promise(resolve => {
+    setTimeout(() => resolve()
+        , timeInMS)
+});
+
+const run = async () => {
+    console.log('Starting simulation...');
+    const liskClient = await getIPCClient('srs-app', '~/.lisk');
+
+    const { tx: transferTrx1 } = await createTransferTrx({
+        moduleID: 2,
+        assetID: 0,
+        asset: {
+            amount: BigInt(108489300000000),
+            recipientAddress: Buffer.from('9cabee3d27426676b852ce6b804cb2fdff7cd0b5', 'hex'),
+            data: '',
+        },
+        passphrase: 'peanut hundred pen hawk invite exclude brain chunk gadget wait wrong ready',
+        fee: '200000',
+        liskClient,
+    });
+
+    try {
+        const { data, status } = await axios.post(getURL('/api/transactions'), transferTrx1);
+        console.log('Transfer transaction 1 is done', data, status);
+    } catch (error) {
+        console.error('Error occurred', error.status);
+        return;
+    }
```

Comment:
u can use the liskClient to send transaction as well, also u can avoid the plugin to have all the controller, it might take a lot of your time, but Ill leave that cal you.

Reply:
this is work in progress where I wanted to use it for tests, maybe in future we can change it